### ISSUE TITLE
"Big Box" Achievements Awarded Prematurely

### ISSUE THREAD

**Issue by Geardan** [larsiusprime/tdrpg-bugs#1081]

**Comment by Geardan** [comment]
I just got both "Big Box o' Killing Stuff" and "Big Box o' Not Dying" Steam achievements before I should have.

Win 10 64bit Steam v2.0.1
Played years ago, got many Steam achievements then, lost that save game.
Have recently played much of NG and NG+ on non-Heroic Mode.
Happened while playing w/ Heroic Mode save file.

1. Beat NG.
2. Got many (but not all) unique armor and weapons.
3. Played NG+.
4. Upgraded most (not all) of said items.
5. Went back to NG, got a unique piece of armor (I think it was Assassin's Folly or Master of Archers). Got awarded both in-game achievements Swiss Army Breastplate and Big Box o' Not Dying. Was also awarded with the Steam achievement for Big Box (already had Swiss Army from years ago).
6. Finished last mission on Extreme difficulty. Got rewards, then same as with armor: Swiss Army and Big Box in-game achievements, and Big Box Steam achievement awarded.

**Comment by PaladinOne** [comment]
I encountered this as well, without even starting a NG+

-Had not beat NG, thus not even started NG+
-Had acquired many (but not all) unique armor and weapons much earlier
-On acquiring a new piece of unique armor (Traitor's Harness?) and weapon (Executioner's Bludgeon? Captain's Warbow?), both Swiss Army and Big Box achievements were granted both in-game, and in Steam itself

**Comment by Dakmordian** [comment]
I actually had the same thing happen to me during the normal game, too. I think I was only about 3/4 of the way through, so I hadn't even had the chance to obtain all gear.

**Comment by stoz** [comment]
I had this happen to me as well. I imported my game from the original version into the new HD version and they triggered as soon as I opened the save (specifically NewGame+ but it was the first one I opened). I had only upgraded roughly half of the unique armours and weapons.

I can send through my exported original save if it helps (was moving between computers as well as upgrading to the new version).

I will admit I am slightly annoyed as I cannot remove these achievements from my steam profile now that they have triggered.
